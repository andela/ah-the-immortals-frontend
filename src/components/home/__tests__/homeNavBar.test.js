import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NavigationBar from '../homeNavbar';

describe('Tests navigation Bar', () => {
  it('Tests navigation Bar component', () => {
    const wrapper = shallow(
      <NavigationBar showModal={jest.fn()} />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
