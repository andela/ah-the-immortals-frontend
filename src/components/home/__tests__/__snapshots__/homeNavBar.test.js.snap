// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests navigation Bar Tests navigation Bar component 1`] = `
<div>
  <Bootstrap(undefined)
    className="homenav fixed-top"
  >
    <NavbarBrand>
      <b
        className="brand"
      >
        Authors Haven
      </b>
    </NavbarBrand>
    <NavbarCollapse>
      <Nav
        className="justify-content-end"
        fill={false}
        justify={false}
        style={
          Object {
            "width": "100%",
          }
        }
      >
        <NavItem>
          <Button
            active={false}
            className="buttons"
            disabled={false}
            onClick={[MockFunction]}
            type="button"
            variant="danger"
          >
            Join now
          </Button>
        </NavItem>
        <NavItem>
          <Button
            active={false}
            className="buttons"
            disabled={false}
            type="button"
            variant="outline-light"
          >
            Sign In
          </Button>
        </NavItem>
      </Nav>
    </NavbarCollapse>
  </Bootstrap(undefined)>
</div>
`;
